tests/unit-tests/tests-give.php

```diff
@@ -15,13 +15,6 @@ public function tearDown() {
 		parent::tearDown();
 	}
 
-	/**
-	 * @covers Give
-	 */
-	public function test_give_instance() {
-		$this->assertClassHasStaticAttribute( '_instance', 'Give' );
```

Comment:
With the refactor to a Service Container, the static instance has been moved out of the Give class.